coalib/bearlib/aspects/collections.py

```diff
@@ -16,6 +16,13 @@ def __init__(self, seq=()):
         list.__init__(self, seq)
 
     def __contains__(self, aspect):
+        if hasattr(self, 'bear') and isinstance(aspect, aspectbase):
+            compatible = False
+            for language in self.bear.LANGUAGES:
```

Comment:
We also have to extend `coalib.bears.meta.bearclass` for making this work properly. It might somehow work now this way, but `aspect.language` is a `coalib.bearlib.languages.Language.Language` instance, and we want to match it with `Language` instances the bear supports. Old-style `bear.LANGUAGES` will never hold `Language` instances. So I propose to add an additional `aspects=`-accompanying meta keyword-only `languages=` attribute to `bearclass` for adding `Language` instances to a bear and store them as lower-case `cls.languages` attribute